dist/css/extent.css

```diff
@@ -395,10 +395,10 @@ nav ul i {
     .test-steps, .category-tests, .exception-tests, .test-content > .node-list {
         margin-top: 30px;
     }
-    .test-steps th:first-child, tr.log > td:first-child, .node-steps th:first-child, .node-steps td:first-child {
+    .test-steps th:first-child, tr.log > td:first-child, .node-steps > .table-results > thead > tr > th:first-child, .node-steps > .table-results > tbody > tr > td:first-child {
```

Comment:
No this will not work, because the `.node-steps > table > td:first-child` will select all the 1st `td` elements whose parent is `table`. But in our case the parent is `tr`, not `table`. That is why the chaining here.